roaring/btree.go

```diff
@@ -379,27 +374,29 @@ func (t *tree) find(q interface{}, k uint64) (i int, ok bool) {
 		for l <= h {
 			m := (l + h) >> 1
 			mk = x.x[m].k
-			switch cmp := t.cmp(k, mk); {
-			case cmp > 0:
+			// note semicolon; the condition is the unspecified implicit true
```

Comment:
looks like this comment shouldn't be here any more

Reply:
yeah, that was from an intermediate form where i had inlined the comparison, so it was `switch cmp := k - mk; {`